todo_app/app.py

```diff
@@ -42,9 +45,10 @@ def add_item():
         card_title=request.form.get('item_name')
         card_desc=request.form.get('item_desc')
         card_due=request.form.get('item_due')
+        card_lastActivity=datetime.datetime.utcnow()
         if not card_title == "": 
-            cardDetails={"idList":trelloBaseList,"name":card_title,"pos":"top","desc":card_desc,"due":card_due}
-            updateTrelloCard('', cardDetails, "POST", key, token) 
+            cardDetails={"idList":defaultStatus,"name":card_title,"pos":"top","desc":card_desc,"due":card_due,"dateLastActivity":card_lastActivity}
```

Comment:
Slightly confusing to stick to calling the status "idList" for legacy reasons instead of naming it "status".